frontend/src/app/modules/bim/bcf/bcf-wp-single-view/bcf-wp-single-view.component.ts

```diff
@@ -1,7 +1,7 @@
 import {Component, Input, OnDestroy, OnInit} from "@angular/core";
 import {StateService} from "@uirouter/core";
 import {WorkPackageResource} from "core-app/modules/hal/resources/work-package-resource";
-import { NgxGalleryOptions, NgxGalleryImage } from 'ngx-gallery';
+import {NgxGalleryImage, NgxGalleryOptions} from '@kolkov/ngx-gallery';
 import {HalLink} from "core-app/modules/hal/hal-link/hal-link";
 
```

Comment:
<table data-meta="generated_by_danger">
  <tbody>
    <tr>
      <td>:warning:</td>
      <td width="100%" data-sticky="false"><del><span data-href="https://github.com/opf/openproject/blob/8b9cafdf80bccf77659a858a80a3f3a68e75b533/frontend/src/app/modules/bim/bcf/bcf-wp-single-view/bcf-wp-single-view.component.ts#L6"></span>Please use <code>ChangeDetectionStrategy.OnPush</code> for this component</del></td>
    </tr>
  </tbody>
</table>